### ISSUE TITLE
DataFrame with UTF-8

### ISSUE THREAD

**Issue by EckisWelt** [titan550/bcpy#12]
If I load data from a DataFrame which has foreign characters such as äöü, the import loads garbage.

To fix that you would need in the bcp command option: '-C 65001'.

And in the format_file_builder you would need to remove any collation.

https://docs.microsoft.com/en-us/sql/relational-databases/import-export/create-a-format-file-sql-server?view=sql-server-ver15

I'm not sure how it needs to be done to be still flexible with your default file import without UTF-8.

**Comment by titan550** [comment]
I added [these modifications](https://github.com/titan550/bcpy/commit/b3e8155cfc76e152e15a59b97fb3ed491dc806c4) that should do the job. Please try it and let me know.

```bash
pip uninstall -y bcpy
pip install --no-index git+https://github.com/titan550/bcpy@b3e8155cfc76e152e15a59b97fb3ed491dc806c4
```

**Comment by anibyeol** [comment]
Hi,
pip uninstall -y bcpy
pip install --no-index git+https://github.com/titan550/bcpy@b3e8155cfc76e152e15a59b97fb3ed491dc806c4

Tried installing this version, but the issue still persists.

**Comment by FriiSRS** [comment]
The issue still persists!

**Comment by FriiSRS** [comment]
> Hi,
> pip uninstall -y bcpy
> pip install --no-index git+https://github.com/titan550/bcpy@b3e8155cfc76e152e15a59b97fb3ed491dc806c4
> 
> Tried installing this version, but the issue still persists.

Same for me

**Comment by anibyeol** [comment]
Yep, not resolved yet. Chinese characters are loaded like 'å¸‚æ”¿ç§‘'

**Comment by MVargasP** [comment]
Hello, I also have the same problem, still unsolved?

**Comment by markteunen65** [comment]
In the "utf8" branch it says in binary_callers.py: 
**if utf8:
        bcp_command += ['-c', '-C65001']**
But according to https://bobsql.com/using-sqlbcp-native-format-file-vs-text-file-and-the-bom/ the command should be:
**-c -C 65001**

Maybe that's the solution to the UTF-8 problem?
